components/motor/gpio/motor_encoder.go

```diff
@@ -265,13 +269,23 @@ func (m *EncodedMotor) makeAdjustments(pos, lastPos float64, now, lastTime int64
 		m.state.currentRPM = deltaPos / deltaTime
 	}
 
+	if rpmDebug {
+		m.logger.Info("making adjustments")
+		m.logger.Infof("lastPos: %v, pos: %v, goalPos: %v", lastPos, pos, m.state.goalPos)
```

Comment:
Why not use Debugf for that?

Reply:
I did Infof so that if you select rpmDebug you don't also have to use the regular debug flag. just so it's one change instead of two, but I can change it